sogs/utils.py

```diff
@@ -102,7 +102,7 @@ def bencode_consume_string(body: memoryview) -> Tuple[memoryview, memoryview]:
 
 
 def server_url(room):
-    return '{}/{}?public_key={}'.format(config.URL_BASE, room or '', crypto.server_pubkey_hex)
+    return '{}/r/{}?public_key={}'.format(config.URL_BASE, room or '', crypto.server_pubkey_hex)
```

Comment:
This is going to have to be delayed until it gets supported by Session, unfortunately.

Reply:
Either revert this (and we'll apply it later, once Session is ready), or perhaps make a `config.` global for whether or not we add the `/r` here.